src/brokers/broker-details/components/BrokerDetailsBreadcrumb/BrokerDetailsBreadcrumb.test.tsx

```diff
@@ -71,4 +129,77 @@ describe('BrokerDetailsBreadcrumb', () => {
     fireEvent.click(toggle);
     expect(toggle).toHaveAttribute('aria-expanded', 'true');
   });
+
+  it('should navigate back to PodsList page when click on Apply button in both form and yaml editor', async () => {
+    const comp = render(
+      <MemoryRouter>
+        <BrokerDetailsBreadcrumb
+          name={name}
+          namespace={namespace}
+          podName={podName}
+        />
+      </MemoryRouter>,
+    );
+    await waitForI18n(comp);
+
+    fireEvent.click(screen.getByTestId('broker-toggle-kebab'));
+    fireEvent.click(screen.getByText('edit_broker'));
+
+    expect(navigate).toHaveBeenCalledWith(
+      `/k8s/ns/${namespace}/edit-broker/${name}?returnUrl=${encodeURIComponent(
+        window.location.pathname,
+      )}`,
+    );
+  });
```

Comment:
If this test is testing that the `apply` button of the yaml editor navigates correctly to some other palce, why is it in the `BrokerDetailsBreadcrumb.test.tsx` file and not in the `YamlEditorView.test.tsx` file ?